### PR TITLE
Andy Mockler

### PR DESCRIPTION
**Pull request by asmockler**
# Landing Page

**Name**: Andy Mockler
**Section**: 1
**Group**: 3
### Requirements
- [x] Uses (Bootstrap)[https://getbootstrap.com], either locally or from a CDN
- [x] Home page
  - [ ] A picture of you
- [ ] Resume page
  - [ ] Education
  - [ ] Experience
  - [ ] Skills
- [x] About page
  - [x] 2 paragraphs about yourself
### Questions

**These questions must be completed before the assignment will be graded.**

_What are some advantages of using code from a CDN?_

_What are some disadvantages of using a CDN?_

_What does Bootstrap do?_

_You are working at a company that is looking to revamp their web presence. List some pros and cons of using Bootstrap as the main CSS framework for the site redesign_

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            <span class="icon-bar"></span>
+          </button>
+          <a class="navbar-brand" href="#">Andy Mockler</a>
+        </div>
+
+        <!-- Collect the nav links, forms, and other content for toggling -->
+        <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+          <ul class="nav navbar-nav">
+            <li class="active"><a href="index.html">Home</a></li>
+            <li><a href="about.html">About</a></li>
```
Comment (asmockler) [inline about.html:27]: Shouldn't this `<li>` be active on this page instead of the Home link?

File: about.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+          </ul>
+        </div><!-- /.navbar-collapse -->
+      </div><!-- /.container-fluid -->
+    </nav>
+
+    <div class="container">
+      <h1>About me</h1>
+      <p class="lead">I am a student living in Provo, UT. After years of trying to find my course in life, I discovered the Information Systems program and decided to enroll in its prerequisite courses. The more I progressed through those courses, the more I knew applying technology to solve business problems is what I wanted to do.</p>
+      <p class="lead">In my free time, I like to make and listen to music. My first passion was playing the drums and it occupied the majority of my time in middle school and high school.</p>
+      <p class="lead">After my sophomore year of college, I spent 2 years in Spain. Since returning to the United States, I still enjoy finding time to practice my Spanish and take part in their unique culture. I have travelled to Paris and spent a summer living in Ottawa, Canada. I would love to continue traveling throughout my life.</p>
```
Comment (asmockler) [inline about.html:37]: Great use of the `lead` class!

File: css/index.css [hunk 3]
```diff
@@ -1,7 +1,15 @@
...
+body {
+  font-family: Arial;
+}
+
+.glyphicon {
+  font-size: 40px;
+  margin-bottom: 20px;
+}
+
+.personalpoints {
```
Comment (asmockler) [inline css/index.css:13]: Great start on your CSS! `.personalpoints` is a little hard to read when written like that. Maybe try something like `.personal-points`?

File: index.html [hunk 4]
```diff
@@ -3,9 +3,51 @@
...
+      </div><!-- /.container-fluid -->
+    </nav>
+
+    <div class="container">
+      <h1>Andy Mockler</h1>
+      <h3>Information Systems student in the BYU Marriott School of Management</h3>
+
+      <div class="row personalpoints">
+        <div class="col-sm-4">
+          <i class="glyphicon glyphicon-book"></i> <br>
```
Comment (asmockler) [inline index.html:39]: Love these sections!!

**Comment by asmockler** [comment]
Looks great so far! Maybe try adding some color. Make sure you remember to answer the questions before the assignment due date.

File: about.html [hunk 5]
```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+  <head>
```
Comment (profganderson) [inline about.html:3]: This is a test comment

**Pull request closed without merging**